convert_hf_to_gguf.py

```diff
@@ -3598,6 +3600,10 @@ def parse_args() -> argparse.Namespace:
         "--metadata", type=Path,
         help="Specify the path for an authorship metadata override file"
     )
+    parser.add_argument(
+        "--z", action="store_true",
+        help="Keep output and ambed tensors at F16"
```

Comment:
```suggestion
        help="Keep output and embed tensors at F16"
```

Reply:
about the name of the option, feel free to suggest it.
I just think is silly to use convert then quantize in these cases while convert alone can do the job in one shot.